uuid_test.go

```diff
@@ -246,7 +246,7 @@ func TestCoding(t *testing.T) {
 	if err != nil {
 		t.Errorf("Parse returned unexpected error %v", err)
 	}
-	if data != data {
+	if !bytes.Equal(data[:], uuid[:]) {
```

Comment:
I am unclear of why this is needed.  data and uuid are arrays so they are directly comparable.